capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -18,7 +22,14 @@ export class WikiMediaComponent implements OnChanges, OnInit {
   loading = true;
   body: string;
   urls = new Map();
-  cityName: string;
+  langlinks = new Map();
+  languages: {[language: string]: string} = Languages as {[language: string]: string};
```

Comment:
Let's also make an interface for this type:
export interface LanguageData {
  [language: string]: string;
}